zola.metainfo.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<component type="console-application">
+  <id>org.getzola.zola</id>
+  <icon type="local">docs/static/favicon.ico</icon>
+
+  <name>Zola</name>
+  <summary>A fast static site generator in a single binary with everything built-in</summary>
+  <developer_name>Vincent Prouillet</developer_name>
```

Comment:
@Keats What do you think about this? We could also put "Vincent Prouillet and contributors" or "Zola contributors".

Reply:
I think "Vincent Prouillet and contributors" should be good. That's what is present in the website footer.